components/home-page/newsletter-sign-up/NewsletterSignUp.styles.js

```diff
@@ -50,17 +50,8 @@ export const NewsletterSignUpTextInput = styled(TextInputField)`
   margin-bottom: 20px;
 `;
 
-export const PrivacyPolicyLinkText = styled(P)`
-  color: ${colors.MAUVE};
-  font-size: 0.8em;
-  text-transform: lowercase;
-  text-align: right;
+export const PrivacyPolicyLinkText = styled.div`
```

Comment:
I think it may have pushed down the link text? The top is the Figma and the bottom the preview deploy 

<img width="572" alt="Screen Shot 2021-04-29 at 3 08 53 PM" src="https://user-images.githubusercontent.com/33647880/116605017-e1e46a00-a8fc-11eb-83d8-7dd387c0eb86.png">
<img width="379" alt="Screen Shot 2021-04-29 at 3 09 00 PM" src="https://user-images.githubusercontent.com/33647880/116605021-e3159700-a8fc-11eb-97e2-cb2ab0c4085b.png">